provisioning/transport/mqtt/src/ProvisioningTransportHandlerMqtt.cs

```diff
@@ -334,7 +333,7 @@ private static DeviceRegistrationResult ConvertToProvisioningRegistrationResult(
             var uriBuilder = new UriBuilder(WsScheme, message.GlobalDeviceEndpoint, Port);
             Uri websocketUri = uriBuilder.Uri;
 
-            using var websocket = new ClientWebSocket();
```

Comment:
ClientWebSocketChannel should own the lifetime of ClientWebSocket. It is weird to pass something to it, but dispose it while the passed object's lifetime has not ended.